backend/api/Controllers/InspectionFindingsController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers
+{
+    [ApiController]
+    [Route("inspection-findings")]
+    public class InspectionFindingsController : ControllerBase
+    {
+        private readonly IInspectionService _inspectionService;
+        private readonly ILogger<InspectionFindingsController> _logger;
+        public InspectionFindingsController(
+            ILogger<InspectionFindingsController> logger,
+            IInspectionService inspectionService
+        )
+        {
+            _logger = logger;
+            _inspectionService = inspectionService;
+
+        }
+
+        /// <summary>
+        /// Add a new inspection finding - need isarStepId to run this
+        /// </summary>
+        /// <remarks>
+        /// <para> This query adds a new area to the database </para>
+        /// </remarks>
+        [HttpPost("add-findings")]
+        [Authorize(Roles = Role.Admin)]
+        [ProducesResponseType(typeof(AreaResponse), StatusCodes.Status201Created)]
```

Comment:
`AreaResponse` is not the correct return type for this request. Maybe an `Inspection` or creating an `InspectionFindingResponse` would be better?

Reply:
I believe this is addressed now, hence resolving.